app/models/like.rb

```diff
@@ -1,4 +1,8 @@
 class Like < ApplicationRecord
-  belongs_to :user
-  belongs_to :post
+  belongs_to :author, foreign_key: :author_id, class_name: 'User'
+  belongs_to :post, counter_cache: :likescounter
+
+  def update_post_like_counter
+    post.increment!(:likescounter)
+  end
```

Comment:
- [ ] [OPTIONAL] Please use the `after_save` rails callback here to update the post's likes counter just like I suggested for `posts` and `comments`